SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -2337,7 +2337,7 @@ void CFVMFlowSolverBase<V, FlowRegime>::Friction_Forces(const CGeometry* geometr
           SurfaceViscCoeff.CL[iMarker_Monitoring] += ViscCoeff.CL[iMarker];
           SurfaceViscCoeff.CD[iMarker_Monitoring] += ViscCoeff.CD[iMarker];
           SurfaceViscCoeff.CSF[iMarker_Monitoring] += ViscCoeff.CSF[iMarker];
-          SurfaceViscCoeff.CEff[iMarker_Monitoring] += ViscCoeff.CEff[iMarker];
+          SurfaceViscCoeff.CEff[iMarker_Monitoring] = ViscCoeff.CEff[iMarker];
```

Comment:
Not sure about this line modification, but I replaced += with = to keep the code with respect to Efficiency consistent.

Reply:
I think consistency wise you would divide SurfaceViscCoeff.CL by CD like in the other methods.
Everything else looks good.